investigator/investigator/cve_provided/investigate_cve_provided.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# thoth-investigator
+# Copyright(C) 2020 Sai Sankar Gochhayat
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Logic for handling cve provided message."""
+
+import logging
+from typing import Dict, Any
+
+from ..common import schedule_kebechet_administrator, register_handler
+from ..configuration import Configuration
+from ..metrics import scheduled_workflows
+
+from .metrics_cve_provided import cve_provided_exceptions
+from .metrics_cve_provided import cve_provided_success
+from .metrics_cve_provided import cve_provided_in_progress
+from prometheus_async.aio import track_inprogress, count_exceptions
+from thoth.messaging import CVEProvidedMessage
+from thoth.common import OpenShift
+from thoth.storages import GraphDatabase
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@count_exceptions(cve_provided_exceptions)
+@track_inprogress(cve_provided_in_progress)
+@register_handler(CVEProvidedMessage().topic_name, ["v1"])
+async def parse_cve_provided(cve_provided: Dict[str, Any], openshift: OpenShift, graph: GraphDatabase):
```

Comment:
do we do need `graph` here?

Reply:
Currently, we don't not sure if we we wanna do anything apart from scheduling Kebechet.